components/tooltip/tooltip.js

```diff
@@ -425,6 +425,7 @@ class Tooltip extends RtlMixin(LitElement) {
 		this._removeListeners();
 		window.removeEventListener('resize', this._onTargetResize);
 		clearDismissible(this._dismissibleId);
+		delayTimeoutId = null;
```

Comment:
There's code here that bypasses the delay when you hover on one tooltip and then move to another. But with the unit tests this causes the "delay tests" to fail because the tooltip appears immediately after a previous test run. The fix is to reset the delay bypass when a tooltip is disconnected.